docs/user_docs/spec_file_format.md

```diff
@@ -0,0 +1,35 @@
+# Spec file format
+**Will be rewritten** 
+This product uses CSV for file reading, this means that group definition is separated by a  ",". The spec for this product will have the following group definitions: name, ring quadrant, isNew, moved and description. For the CSV, this will look like:
+```
+name,ring,quadrant,isNew,moved,description
+```
+The reader will then translate these to structs which get translated to types using unmarshalling.
+
+The types foreach group:  
+```  
+- name        : string
+- ring        : string
+- quadrant    : string
+- isNew       : bool
+- moved       : int
+- description : string
+```
+
+Some of the groups is defined to take specific inputs. Groups can have these inputs:
+```
+- name        : The name of a data management, datastore, infrastructure or langauge.
```

Comment:
Would software tools be able to describe these adequately?

Also, I'm confused by what's meant by infrastructure here as an example on a tool-type in this tech radar :')

Reply:
if you are refering to 
> name        : The name of a data management, datastore, infrastructure or langauge.
then I mean language isn't really a software tool imo and infrastructure doesn't have to be either so would say specifying them explicitly works better.

> Also, I'm confused by what's meant by infrastructure here as an example on a tool-type in this tech radar :')
Guessing this could refer to things like cloud based webapp hosts like Azure WebApp deployment but I agree that it is a little vague but that might not be a bad thing as the radar is supposed to sort of allow for a bunch of different tech stuff to all live in the same system

Reply:
> would say specifying them explicitly works better

> the radar is supposed to sort of allow for a bunch of different tech stuff to all live in the same system

I feel like these two statements are contradictory? In terms of the latter, this is also why I suggested a more umbrella-ish term, but couldn’t come up with anything except software tool as in tool stack.

---

> then I mean language isn't really a software tool imo and infrastructure doesn't have to be either

Hmm, I kind of see eg. languages as tools too, whether markup or programming, as it’s a fundamental part or tool to build the software, if that makes sense.

Reply:
I'm indifferent to changing it so if other people are fine with changing it then feel free to do so :)

Reply:
External discussion: Minor language change, does not hinder comprehension of the desc. If it should down the line, it will be updated in a seperate ticket in a future sprint.

Reply:
Some of the wording, should indeed be changed and cleaned up. I think this should be worked on as we go on with the project.